cmd_tools/create_sdk.py

```diff
@@ -109,27 +109,32 @@ def copy_rif_sdk():
             shutil.copy(str(lib), str(sdk_lib_dir))
 
     elif OS == 'Linux':
-        shutil.copy(str(bin_dir / "libRadeonImageFilters.so.1.6.1"),
+        shutil.copy(str(bin_dir / "libRadeonImageFilters.so.1.7.0"),
```

Comment:
This seems like it could be made more generic no? So we don't need to update this version with each release?

Reply:
yes, will do